addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -17,6 +17,7 @@ class StockPickingBatch(models.Model):
     name = fields.Char(
         string='Batch Transfer', default='New',
         copy=False, required=True, readonly=True)
+    description = fields.Char("Description", size=35)
```

Comment:
I would avoid the size limit.
    size = None                         # maximum size of values (deprecated)